llvm/lib/Target/RISCV/GISel/RISCVCallLowering.cpp

```diff
@@ -336,6 +357,10 @@ static bool isSupportedReturnType(Type *T, const RISCVSubtarget &Subtarget) {
     return true;
   }
 
+  if (T->isVectorTy() && Subtarget.hasVInstructions() && T->isScalableTy() &&
```

Comment:
Isn't `isSupportedReturnType` used by lowerCall too?

Reply:
I think that’s a different function.

Reply:
What's a different function?

isSupportedReturnType has 2 callers just like isSupportedArgumentType. You're only testing one of the callers.

Reply:
My bad. Updated.